platform/nodejs/package.json

```diff
@@ -22,5 +22,9 @@
   "bugs": {
     "url": "https://github.com/gorhill/uBlock/issues"
   },
-  "homepage": "https://github.com/gorhill/uBlock#readme"
+  "homepage": "https://github.com/gorhill/uBlock#readme",
+  "dependencies": {
```

Comment:
Note that these dependencies are a bad idea. Ideally the benchmark package should do all of this on its own. But in the interest of moving things forward it's OK for now.

Reply:
I was happy to not have external dependencies... Maybe eventually this could be a special make target which would patch package.json to add those dependencies -- not sure if this makes sense

Reply:
I don't think this bundle should be provided at all in the end. I'll try to do this in the benchmark repo and if it works out alright then this stuff can be removed.